### ISSUE TITLE
Elements deleted from Scenes view when changing parent ID

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#2041]
From a [user post](https://groups.google.com/d/msg/maqetta-users/0bnV52xDR1k/zvbrv3j12YsJ):

> After adding a checkbox or a text area (using dojox.mobile elements) into a div, if the div ID is changed (or even if simply set), the added elements are deleted. 

I assume this refers to the **Scenes** view.

**Comment by JonFerraiolo** [comment]
I tried to reproduce this in M5 (and latest code) and I'm able to change IDs on DIVs that contain mobile widgets, and the mobile widgets don't disappear. There must be something else going on that's not listed in the original report.

Reassigning to @jhpedemonte to see if he can come up with a reproducible error. Otherwise, maybe close this one and wait until we have more detail about how to reproduce.

There is a problem where Scenes view doesn't update after changing an ID in source view. I'm pretty sure that Dartas is doing some source editing. I created #2292 for that.
